src/schema/rules/files/deriv/preprocessed_data.yaml

```diff
@@ -104,6 +104,13 @@ fmap_pepolar_common:
     space: optional
     description: optional
 
+fmap_pepolar_m0scan:
+  $ref: rules.files.raw.fmap.pepolar_m0scan
```

Comment:
@mattcieslak it looks like the problem is that the `dereference` function can't handle multiple levels of references, so since `rules.files.raw.fmap.pepolar_m0scan` has a `$ref` in it as well, it's not being expanded.

Reply:
@effigies that's a bug in `dereference`, right?

Reply:
looks like copy/paste solved it for now

Reply:
That's definitely good, but if it _is_ a bug I'll want to open an issue about it.

Reply:
Likely what happened was that references of objects with references in them do not get dereferenced enough times. So that probably is a bug.